.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
```

Comment:
we don't have any tags, should we just remove that?

Reply:
we could, i'm wondering if we should have tags though? that respond to the rasa versions that it works on? Or do we always want to push the latest?

Reply:
i would say just do latest for now, we can always change later